### PR TITLE
Include more reservedPaths. Fixes #1

### LINKED ISSUE #1: Reserved paths
It looks like you're missing a few reserved paths (user/repo names):

```js
const reservedPaths = [
  "about",
  "access",
  "account",
  "admin",
  "blog",
  "business",
  "contact",
  "dashboard",
  "developer",
  "events",
  "explore",
  "features",
  "gist",
  "integrations",
  "issues",
  "join",
  "login",
  "marketplace",
  "mirrors",
  "new",
  "notifications",
  "open-source",
  "organizations",
  "orgs",
  "pages",
  "personal",
  "pricing",
  "pulls",
  "search",
  "security",
  "settings",
  "showcases",
  "site",
  "stars",
  "styleguide",
  "support",
  "trending",
  "watching"
];
```

If you're interested, check out my [github-reserved-names](https://github.com/Mottie/github-reserved-names) npm package. It also includes a bunch of reserved numbered names (e.g. `https://github.com/404`) which may not be as important to include.

### PR DESCRIPTION
**Pull request by Mottie**
I ran `npm test` and found these messages which I did not tackle in this PR:

```
index.js
  ‼   66:1   Function shortenURL has a complexity of 22.  complexity
  ‼  146:29  Do not nest ternary expressions              no-nested-ternary
```

### CODE REVIEW

**Comment by Mottie** [comment]
Oh LOL

File: index.js [hunk 1]
```diff
@@ -6,33 +6,7 @@ const labelRegex = /labels[/]([^/]+)/;
...
-	'personal',
-	'security',
-	'dashboard',
-	'showcases',
-	'open-source',
-	'marketplace',
-	'organizations',
-	'notifications',
-];
+const reservedPaths = require('github-reserved-names').all;
```
Comment (sindresorhus) [inline index.js]: `require` should be at the top to make it easier to switch to `import` in the future.

Comment (Mottie) [inline index.js]: Ok

**Comment by bfred-it** [comment]
Looking good! Do you mind requiring the JSON directly? It saves a require trip

**Comment by bfred-it** [comment]
`const reservedPaths = require("github-reserved-names/reserved-names.json");`

**Comment by Mottie** [comment]
LOL cool... so many ways to do the same thing ;)

**Comment by Mottie** [comment]
LOL just as you merge, I found another reserved name... I'll publish it in a minute.

**Comment by bfred-it** [comment]
If it matches `^1.0.3` it'll be in automatically :)

**Pull request merged**